filebeat/Makefile

```diff
@@ -7,6 +7,10 @@ TEST_ENVIRONMENT?=true
 GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
 ES_BEATS?=..
 
+TARGETS?="windows/amd64 windows/386 darwin/amd64"
+TARGETS_OLD?="linux/amd64 linux/386"
+CGO=true
```

Comment:
Ok, I was on the fence about it, so I changed the PR to make `CGO=true` the default in all Beats, and overwrite it to `false` in Winlogbeat.